cli/cmd/env.go

```diff
@@ -125,6 +129,15 @@ var _envListCmd = &cobra.Command{
 			exit.Error(err)
 		}
 
+		if _flagOutput == flags.JSONOutputType {
+			bytes, err := libjson.Marshal(cliConfig.Environments)
```

Comment:
Should we use `MarshalIndent()` here to make it look nicer?

Reply:
As discussed, for now `-o json` will not pretty print json.